### PR TITLE
Update R4 Encounter create to include custom-attribute examples for integer and date_time value types.

### PR DESCRIPTION
**Pull request by exopsRamba**
Description
----
Screenshots before the merge: 
Screen Shot 2022-04-05 at 5 22 29 PM
Screen Shot 2022-04-06 at 10 50 47 AM
Screen Shot 2022-04-05 at 5 22 44 PM

Screen Shot 2022-04-06 at 3 39 57 PM

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: lib/resources/r4/encounter.yaml
Comment (alexbeckner1) [inline lib/resources/r4/encounter.yaml:609]: I thought Ignite typically capitalizes the data types based on how they are represented in the spec here: https://www.hl7.org/fhir/datatypes.html#CodeableConcept

Comment (alexbeckner1) [reply lib/resources/r4/encounter.yaml:609]: I also think we need periods at the end of these notes.

Comment (ahamdany) [reply lib/resources/r4/encounter.yaml]: https://github.com/cerner/fhir.cerner.com/pull/825/commits/a5347f8fbcc2881fe55f67f9f40f32a631117096

**Review by Anurag-Balannagari — APPROVED** [review]

**Review by ahamdany — APPROVED** [review]

File: lib/resources/r4/encounter.yaml
Comment (sydneanschutz) [inline lib/resources/r4/encounter.yaml:578]: nit: single quotes

**Comment by sydneanschutz** [comment]
LGMT, I'll approve after i see the rebased version after this is merged though https://github.com/cerner/fhir.cerner.com/pull/812

File: lib/resources/r4/encounter.yaml
Comment (exopsRamba) [reply lib/resources/r4/encounter.yaml]: It looks like we use double quotes throughout this file and other resource yaml files.

File: lib/resources/r4/encounter.yaml
Comment (sydneanschutz) [inline lib/resources/r4/encounter.yaml:632]: string, integer, and dateTime are lower case bc they're primitive types

**Review by sydneanschutz — APPROVED** [review]

File: lib/resources/r4/encounter.yaml
Comment (ahamdany) [reply lib/resources/r4/encounter.yaml]: updated https://github.com/cerner/fhir.cerner.com/pull/825/commits/1f51d6f592da1b972737f18b7ab944114ebfe8b9

**Pull request merged**
